src/mapping.ts

```diff
@@ -46,6 +46,7 @@ function createPool(poolId: string) : void {
   pool.totalRepaysAggregatedAmount = BigInt.fromI32(0)
   pool.totalBorrowsCount = 0
   pool.totalBorrowsAggregatedAmount = BigInt.fromI32(0)
+  pool.version = BigInt.fromI32(poolMeta.version == 2 ? 2 : 3)
```

Comment:
would always use the strict equality operator ===

Reply:
This is because of https://www.assemblyscript.org/basics.html#triple-equals